sr_blockly_blocks/src/sr_blockly_blocks/web_service_module.py

```diff
@@ -12,9 +12,7 @@ def get_named_poses(self, parameters):
             for robot_name in parameters:
                 response = list_states('', robot_name)
                 for state in response.states:
-                    full_state_name = state + ' (' + robot_name + ')'
-                    identifier_name = full_state_name.lower().replace(' ', '_').replace('(', '_').replace(')', '_')
-                    result.append([full_state_name, identifier_name])
+                    result.append(state)
```

Comment:
@kirstyellis as parameter this method is taking robot names list. What would you do in case if two robots has the same state name?